ams/server/src/main/java/com/netease/arctic/server/ArcticServiceContainer.java

```diff
@@ -301,57 +302,72 @@ private class ConfigurationHelper {
     private JSONObject yamlConfig;
 
     public void init() throws IOException {
-      initServiceConfig();
+      Map<String, Object> envConfig = initEnvConfig();
+      initServiceConfig(envConfig);
       initContainerConfig();
```

Comment:
What's the order of ENV config and configs in file? The file config has higher priority ?

Reply:
env variable is larger than the configuration file.When the env is configured, the file configuration properties does not take effect.If configured, it will be overwritten by environment variables. what do you think ?

Reply:
Have you tested the code? It looks like properties in config file will overwrite the envProperties.